act_dr6_lenslike/act_dr6_lenslike.py

```diff
@@ -135,6 +136,41 @@ def parse_variant(variant):
     include_planck = True if 'actplanck' in variant else False
     return v,baseline,include_planck
 
+class bin1D:
```

Comment:
I'd revert this back to an orphics import (I try to avoid having code duplicates floating around).  I know orphics is not a standard code and not on pip, but the class in this module that calls orphics and pyfisher are not intended for public use.